docs/application/dotnet/guides/machine-learning/overview.md

```diff
@@ -0,0 +1,20 @@
+# Machine Learning
+
+The Machine Learning Inference feature allows developers to invoke neural network models with less efforts and higher efficiency.
+
+With Tizen.MachineLearning.Inference.SingleShot, you can load the existing neural network model or your own specific model from storage and invoke it with a single instance of input data, finally get the inference output result.
+
+The feature of Tizen.MarhineLearning.Inference.Pipeline allows to manage the topology of
```

Comment:
@Reni-90 As this page is overview.md, in there, simplified sentences are better. 
For here, I think we don't need to apply \` to Tizen.MarhineLearning.Inference.Pipeline. 
Or "You can also use Pipeline feature. This feature..." would be also enough, in my thought.

@again4you As this paragraph is talking about a feature not added yet, how about moving to line 17?

For example : 

---

# Machine Learning

The Machine Learning Inference feature allows ...
 
You can use the following machine learning feature in your .NET applications:

 - [SingleShot](singleshot.md)
         You can use the SingleShot class, to load the existing neural network model or your own specific model from the storage...

You can also use Pipeline feature.... This feature will be available from the next Tizen version.

## Related Information
...

Reply:
@mijin85cho Your idea look good for me. I will.

Reply:
It is also possible. 

> **Note**
>
> You can also use Pipeline feature.... This feature will be available from the next Tizen version.

"**Related Information**" is working as a kind of footer for every page, so please put the note before the "**Related Information**" section.

Reply:
@again4you It is also possible.

> **Note**
> 
> You can also use Pipeline feature.... This feature will be available from the next Tizen version.

"**Related Information**" is working as a kind of footer for every page, so please put the note before the "**Related Information**" section.

Reply:
Is there a need to put the pipeline feature in the note tag? 
Can we slightly change to:
# Machine Learning
Machine Learning...

You can use the following machine learning features in your .NET applications:

- [SingleShot](singleshot.md)
    You can use the SingleShot feature to load the existing model, invoke it, and get the inference output result.
- [Pipeline]()
    You can use the Pipeline feature to manage the topology of data and the interconnection between processors and models.

    >**Note**
    >
    >This feature is available in Native APIs from Tizen 5.5. However, this feature is 
       not available in .NET APIs. This feature will be available in the .NET APIs from the 
       next Tizen version.
   
## Related Information
- Dependencies
    -   Tizen 5.5 and Higher



@mijin85cho and @again4you 
Please verify.
We can rephrase the pipeline definition if needed and we can also add the link for the pipeline document if possible. 
Please correct me if I am wrong.

Reply:
@Reni-90 The problem is as written in the note&comment itself, **the Pipeline is not available in .NET API**.
So we don't have document for Pipeline for .NET.
Please check that we are reviewing .NET guide page. We don't have Pipeline for tizen 5.5 .NET APIs, and it will be supported from the next version.

Reply:
@mijin85cho 
My intention was that we can add the document when available (in the next Tizen version we can update the document link) and call out pipeline as a feature. 

- Pipeline
    You can use the Pipeline feature to manage the topology of data and the interconnection between processors and models.

    >**Note**
    >
    >This feature is available in Native APIs from Tizen 5.5. However, this feature is 
       not available in .NET APIs. This feature will be available in the .NET APIs from the 
       next Tizen version.

As the note is aligned to **Pipeline**, it will be clear that the feature is not available in .NET, so I think calling out pipeline as a feature is better than putting it in a note tag.
Please guide me if I am missing any points here.

Reply:
Good!

Reply:
@again4you
Thank you very much.
I think we can use this format, once the pipeline feature is added.
As per @mijin85cho's suggestion, it is good to use the format once the feature is added.
Therefore, can we have the following slight change here according to https://github.com/Samsung/tizen-docs/pull/891#discussion_r337359607:

# Machine Learning
Machine Learning...

You can use the following machine learning features in your .NET applications:

- [SingleShot](singleshot.md)
    You can use the SingleShot feature to load the existing model, invoke it, and get the inference output result.

You can also use the Pipeline feature to manage the topology of data and the interconnection between processors and models. This feature is available in Native APIs from Tizen 5.5. However, this feature is not available in .NET APIs. This feature will be available in the .NET APIs from the next Tizen version.
   
## Related Information
- Dependencies
    -   Tizen 5.5 and Higher


Please verify and share your opinion.